dist/index.js

```diff
@@ -385,8 +385,15 @@ class Action {
             + "<tr><td>Action</td><td>Minimal</td><td>Low</td><td>Medium</td><td>High</td><td>Criticalℹ️</td><td>Unknown</td>"
             + "<td>Result</td></tr></thead><tbody>";
         for (const task of report.actions) {
-            task.passed ? tasksPassed++ : tasksFailed++;
-            if (task.tests) {
+            if (task.passed !== undefined || task.passed !== null) {
+                if (task.passed === true) {
+                    tasksPassed++;
+                }
+                else {
+                    tasksFailed++;
+                }
+            }
+            if (task.tests && task["passed"] === true) {
                 core.info(`${ansi_colors_1.default.bold(`${task.action_id} action:`)} ${task.passed === true ? ansi_colors_1.default.green("passed") : ansi_colors_1.default.red("failed")} » `
```

Comment:
There is repetition in this line of code; we should avoid duplicating code

Reply:
Also we are checking if `task.passed === true` twice. Maybe you should create a variable with that content and refer to it here. WTDY?

Reply:
I'm going to merge the PR to fix the 'passed' field issue and I will create another task to improve that part the code. Thanks for your suggestion 😄